libs/shared/e2e-utils/src/lib/utils.ts

```diff
@@ -105,3 +110,57 @@ export async function waitFor(ms: number) {
 export function isWindows() {
   return process.platform === 'win32';
 }
+
+export async function createInvokeMCPInspectorCLI(
+  e2eCwd: string,
+  mcpProjectName = 'nx-mcp',
+) {
+  const graph = await createProjectGraphAsync();
+  const nxMcp = graph.nodes[mcpProjectName];
+  if (
+    !nxMcp ||
+    !nxMcp.data.targets?.build?.options?.outputPath ||
+    !nxMcp.data.targets.build.options.outputFileName
+  ) {
+    throw new Error('NX MCP project not found');
```

Comment:
incredible nit but if the name is configurable then the error message could be to